modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APIEndpointRelevantObjectEntryModelListenerTest.java

```diff
@@ -173,7 +202,7 @@ public void test() throws Exception {
 			).toString(),
 			"headless-builder/schemas", Http.Method.POST);
 
-		jsonObject = HTTPTestUtil.invokeToJSONObject(
+		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
```

Comment:
We could check here also with the `JSONAssert.assertEquals` method with a different `JSONCompareMode`, for example:

```
		JSONAssert.assertEquals(
			JSONUtil.put(
				"r_apiApplicationToAPIEndpoints_c_apiApplicationId",
				apiApplicationJSONObject.get("id")
			).put(
				"status", JSONUtil.put("code", 0)
			).toString(),
			HTTPTestUtil.invokeToJSONObject(
				JSONUtil.put(
					"httpMethod", "get"
				).put(
					"name", RandomTestUtil.randomString()
				).put(
					"path",
					StringPool.FORWARD_SLASH + RandomTestUtil.randomString()
				).put(
					"r_apiApplicationToAPIEndpoints_c_apiApplicationId",
					apiApplicationJSONObject.getLong("id")
				).put(
					"r_requestAPISchemaToAPIEndpoints_c_apiSchemaId",
					apiSchemaJSONObject.getLong("id")
				).put(
					"r_responseAPISchemaToAPIEndpoints_c_apiSchemaId",
					apiSchemaJSONObject.getLong("id")
				).put(
					"scope", "company"
				).toString(),
				"headless-builder/endpoints", Http.Method.POST
			).toString(),
			JSONCompareMode.LENIENT);
```